Documentation/HowToUseETW.md

```diff
@@ -0,0 +1,46 @@
+# Supported
+ETW logs are available on CoreCLR only currently.
+
+
+# How to Collect ETW logs use logman
+
+
+## Start the logging
+
+ * Option 1: Generate a single file that does circular logging with Verbose level, For example
+ 
+ ```
+logman start WCFETWTracing -p "Microsoft-Windows-Application Server-Applications" 0xFFFFFFFF  0x5 -bs 64 -nb 120 320 -ets -ct perf -f bincirc -max 500 -o c:\temp\WCFEtwTrace.etl
```

Comment:
I recommend you also add a statement saying on which machine you execute this command.  For example, when we run the Bridge on a separate machine, is this the command you use on the client machine or the server machine?  When the client and server are on different machines, would you use this same command on both machines to collect ETW event for both client and server independently? When client and server are on the same machine, would this make the ETW events be a combination of client and server events?

Also, please say when this command can be entered.  For example, must you do it before you start the Bridge?  Can you do it after the Bridge is already running?  If you stop and restart the Bridge, do you need to enter it again?